src/renderer/browser/components/RequestsProcessor/TestInvoke/index.js

```diff
@@ -40,7 +40,7 @@ export default function makeStorageComponent(testInvokeActions) {
     })),
     withNullLoader(testInvokeActions),
     withRejectMessage(testInvokeActions, (props) => (
-      `Invocation failed for operation "${props.operation}" on "${props.scriptHash}"`
+      `Invocation failed for operation "${props.operation}" on "${props.scriptHash}"\n${props.error}`
```

Comment:
We might want to destructure this as well, just for the sake of keeping things consistent :)